server/game/core/Constants.ts

```diff
@@ -35,6 +35,13 @@ export enum StatType {
     Power = 'power'
 }
 
+export enum DamageType {
+    Combat = 'combat',
+    Ability = 'ability',
+    Excess = 'excess',
+    Overwhelm = 'overwhelm'
+}
```

Comment:
I moved this here because importing `DamageSystem` in `CostLibrary` was causing an import cycle.

Reply:
Ugh yeah I hate that that's an issue, glad you were able to resolve it. Thanks for the note